GOFevaluation/evaluator_base.py

```diff
@@ -81,10 +84,13 @@ def from_binned(cls, binned_data, binned_reference):
 
     @classmethod
     def bin_equiprobable(cls, data_sample, reference_sample, nevents_expected,
-                         n_partitions, order=None, plot=False, **kwargs):
+                         n_partitions, order=None, plot=False,
+                         plot_mode='sigma_deviation', **kwargs):
         """Initialize with data and reference sample that are binned
         such that the expectation value is the same in each bin.
-        kwargs are passed to `plot_equiprobable_histogram` if plot is True.
+        kwargs are passed to `plot_equiprobable_histogram`
+        if plot is True.
+        'plotmode' can be set to num_counts or count_density
```

Comment:
I would prefer not to put the `plot_mode` kwarg here. It would be better to just catch it as a `**kwarg` within the plotting function. In principle I would like people to use the plotting function and not this init to do anything fancier than getting a first quick look